platform/src/test/kotlin/org/stellar/anchor/platform/LogAppenderTest.kt

```diff
@@ -0,0 +1,24 @@
+package org.stellar.anchor.platform
+
+import com.google.gson.*
+import java.io.*
+import org.junit.jupiter.api.*
+import org.junit.jupiter.api.Assertions.*
+import org.stellar.anchor.util.*
+
+class LogAppenderTest {
+  @Test
+  fun test_logJsonAppenderFormat() {
+    val outputStreamCaptor = ByteArrayOutputStream()
+    System.setOut(PrintStream(outputStreamCaptor))
+    Log.info("hello world")
+
+    val json = outputStreamCaptor.toString().trim()
+    val gson = Gson()
+
+    assertDoesNotThrow { gson.fromJson(json, JsonLog::class.java) }
+    val actualLog = gson.fromJson(json, JsonLog::class.java)
```

Comment:
You can use optionals to avoid running the same command twice, as we do here:
https://github.com/stellar/java-stellar-anchor-sdk/blob/f82985c0ce81df8aeee6fd3c95db30a224a7e69e/core/src/test/kotlin/org/stellar/anchor/sep38/Sep38ServiceTest.kt#L304-L307